### ISSUE TITLE
[DOC]: TransportServer and VirtualServer snat field description

### ISSUE THREAD

**Issue by alonsocamaro** [F5Networks/k8s-bigip-ctlr#3271]
#### Setup Details
 CIS Version : 2.15.1      
 Build: f5networks/k8s-bigip-ctlr:latest       
 
#### Description  
 
The current documentation pages of VirtualServer and TransportServer for the snat field miss the option "auto":

```
Reference to SNAT pool on BIG-IP. The other allowed value is: none.
```

Unlike the examples, which do use the option.

**Comment by trinaths** [comment]
Created [CONTCNTR-4386] for internal tracking

**Comment by trinaths** [comment]
Fixed in 2.16
